dti-website/src/data/all-members.json

```diff
@@ -6,7 +6,7 @@
     "major": "CS",
     "hometown": "Manhattan, KS",
     "about": "Hi! I'm a CS and Math double major in the college of A&S. My main interests are in web dev, data science, and AI/ML. I love working on creative products that positively impact people's lives. In my free time I like playing league of legends, watching anime, and hanging out with friends.",
-    "subteam": "reviews",
+    "subteam": "",
```

Comment:
Revert changes in this file unless the data is good.

Reply:
Can I ask how I can check whether the data is good? I didn't make these changes, so I'm not quite sure where to look.

Reply:
@gracshan The current data about subteam should be mostly right, so diffs that change subteam field to empty string is definitely wrong. We cannot merge this PR because once we merge it, the team website with the wrong data will be auto-deployed.

Reply:
@SamChou19815 I see. I'll revert it based on what I see displayed in the changes here. Thank you for letting me know!